static/sass/components/_project-header.scss

```diff
@@ -0,0 +1,34 @@
+// Project Case Studies Header
+// ===========================
+
+
+.project-meta {
+  @include center('medium-page');
+  align-items: flex-end;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: size('gutter');
+
+  @include above('full-content') {
+    justify-content: space-between;
+  }
+}
+
+.project-title {
```

Comment:
It might be a new story, but I think this pattern needs to be fleshed out in a re-usable way. Maybe the entire meta-section should be (including featured links).